docs/settingUp.md

```diff
@@ -1,333 +1,86 @@
-`Staging server` is the server instance you set up on Google App Engine for hosting the app for testing purposes.
-
-1. Create your own app on GAE.
-    Suggested app identifier: `teammates-yourname` (e.g. `teammates-john`).<br> 
-    The URL of the app will be like this. `http://teammates-yourname.appspot.com`
-
-2. Modify configuration files.
-   * `src/main/resources/build.properties` <br>
-      Edit the file as instructed in its comments.
-   * `src/test/resources/test.properties` <br>
-      Edit the file as instructed in its comments. 
-   * `src/main/webapp/WEB-INF/appengine-web.xml`<br>
-      Modify to match app name and app id of your own app.
-      
-3. Deploy the application to your staging server.
-   * Choose `Deploy to app engine` from eclipse (under the `Google` menu item ![](https://cloud.google.com/appengine/docs/python/images/transform_resize_after.jpg) ) and follow the steps.
-   * Wait until you see this message in Eclipse console `Deployment completed successfully`
-   * Go to appengine dashboard `https://appengine.google.com/dashboard?&app_id=teammates-name`
-   * Click `Versions` under `Main` menu on the left bar.
-   * Set the version you deployed as the `default`. <br>
-   Note: You can skip the steps to set the deployed version as the default. 
-   In that case, you can access the deployed app using 
-   `http://{version}-dot-{app-id}.appspot.com` e.g. `http://4-18-dot-teammates-john.appspot.com`
-    You can run the tests again against the deployed app 
-    (modify `test.properties` so that tests execute against the 
-    deployed app and not the dev server).
-    Note that GAE daily quota will be exhausted after 2-3 runs of the full test suite.
-
-
-## Running client scripts
-Client scripts are scripts that remotely manipulate data on GAE via its Remote API. Most of developers may not need to write and/or run client scripts but if you are to do so, additional steps are required:
-
-1. Download and install Google Cloud SDK at https://cloud.google.com/sdk/downloads.
-
-2. Run `gcloud auth login` in the terminal of your PC and choose your google account for authentication.
-
-3. Now you can run your scripts.
+1. Modify the following config files:
+   * `gradle.properties`<br>
+      If you want to use a JDK other than the one specified in your PATH variable, add the value to the variable `org.gradle.java.home`.
+      This value must be a valid **JDK 1.7** directory.
+   * `src/test/resources/test.properties`<br>
+      Append a **same** unique id (e.g your name) to **each** of the default accounts found at the bottom of this file,
+      e.g change `test.student1.account=alice.tmms` to `test.student1.account=alice.tmms.KevinChan`.
 
+## Step 4: Set up Eclipse IDE
 
-## Troubleshooting
-Troubleshooting instructions are given [in this document](troubleshooting-guide.md)
+Eclipse IDE is our preferred development environment and support request for TEAMMATES-specific Eclipse issues will be entertained.
```

Comment:
Actually my intention here was that support request for non-Eclipse issues will not be entertained. I feel like it needs to be surfaced because our current workflow is very much IDE-independent (Travis CI _is_ IDE-independent) and using IntelliJ (I've seen at least 3 contributors trying to set up with IntelliJ) or Netbeans or other IDEs _might_ be possible, it's just that we only provide support for Eclipse.
Should I rephrase it to something like "...environment. We do not provide support for other IDEs."?